sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -76,6 +78,23 @@ public void verbose(String message) {
         }
     }
 
+    /**
+     * Logs a message at {@code verbose} log level.
+     *
+     * <p><strong>Code samples</strong></p>
+     *
+     * <p>Logging a message at verbose log level.</p>
+     *
+     * {@codesnippet com.azure.core.util.logging.clientlogger.verbose}
+     *
+     * @param message The message to log.
+     */
+    public void verbose(Supplier<String> message) {
+        if (logger.isDebugEnabled()) {
+            Optional.ofNullable(message).ifPresent(msg -> logger.debug(sanitizeLogMessageInput(msg.get())));
```

Comment:
Could this be simplified to just use `message != null`?

Reply:
Sure, fixed